index.html

```diff
@@ -71,7 +71,7 @@ <h2 id="main-h2">Private Digital Currency</h2>
                         </div>
                         <div class="col-lg-6 col-md-6 col-sm-8 col-xs-12 why-text">
                             <h3>Monero is fungible.</h3>
-                            <p>Because of the on-by-default nature of the privacy technologies, Monero is fungible, meaning one Monero will always be equal to another. This means you won't have to worry about Monero blacklisted by exchanges or vendors.</p>
+                            <p>Privacy is always-on. Therefore Monero is fungible, meaning  Monero will never be blacklisted by exchanges or vendors due to its association in previous transactions.</p>
```

Comment:
> Privacy is always-on.
I think `private by default` has become somewhat of a tagline for Monero. I think we should use it as much as possible if appropriate.

>Therefore Monero is fungible
A leap. Maybe: 
`Because Monero is private by default, every transaction is private. That means that no Monero can be traced back to anything "nefarious". This means that Monero is truly fungible, every single Monero will always be equal to another. This makes it very unlikely that Monero could have be blacklisted.`